### DISCUSSION TITLE
Cygwin errors with `bash: eval: line 80: syntax error: unexpected end of file`

### DISCUSSION THREAD

**Discussion by mybearworld** [JanDeDobbeleer/oh-my-posh#4993] [Troubleshoot]
When I use the following .bashrc file with Cygwin:

```sh
eval "$(oh-my-posh init bash)"
```

I get the following terminal output. The prompt still seems to appear, though.
Do you know why this might be happening?

```
bash: eval: line 80: syntax error: unexpected end of file
```

**Comment by JanDeDobbeleer** [comment]
@mybearworld I'd be surprised that error is present and a successful prompt is shown. Unless the error comes from another tool, do you have this error only when adding the oh-my-posh init line?

**Reply by mybearworld** [reply]
I did have a commented out line there - when I remove that, the error message changes to be at line 79, but it still errors.

**Reply by JanDeDobbeleer** [reply]
@mybearworld is that error also there when you only have the oh-my-posh init line in your `.bashrc`?

**Reply by mybearworld** [reply]
Yes

**Reply by JanDeDobbeleer** [reply]
I can't reproduce this at all.
